sysl2/sysl/Parser_test.go

```diff
@@ -133,239 +142,233 @@ func parseAndCompare(
-	testParseAgainstPython(t, "tests/school.sysl", "")
+	testParseAgainstGolden(t, "tests/school.sysl", "")
 }
 
 func TestImports(t *testing.T) {
-	testParseAgainstPython(t, "tests/library.sysl", "")
+	testParseAgainstGolden(t, "tests/library.sysl", "")
 }
 
 func TestRootArg(t *testing.T) {
-	testParseAgainstPython(t, "school.sysl", "tests")
+	testParseAgainstGolden(t, "school.sysl", "tests")
 }
 
 func TestSequenceType(t *testing.T) {
 	testParseAgainstGolden(t, "tests/sequence_type.sysl", "")
 }
 
 func TestRestApi(t *testing.T) {
-	testParseAgainstPython(t, "tests/test_rest_api.sysl", "")
+	testParseAgainstGolden(t, "tests/test_rest_api.sysl", "")
 }
 
 func TestRestApiQueryParams(t *testing.T) {
-	testParseAgainstPython(t, "tests/rest_api_query_params.sysl", "")
+	testParseAgainstGolden(t, "tests/rest_api_query_params.sysl", "")
 }
 
 func TestSimpleProject(t *testing.T) {
-	testParseAgainstPython(t, "tests/project.sysl", "")
+	testParseAgainstGolden(t, "tests/project.sysl", "")
 }
 
 func TestUrlParamOrder(t *testing.T) {
-	filename := "tests/rest_url_params.sysl"
-	parseAndCompareWithPython(filename, "", false)
-	fmt.Printf("Output for %#v won't match legacy. Visually inspect the above diff.\n", filename)
+	testParseAgainstGolden(t, "tests/rest_url_params.sysl", "")
 }
 
 func TestUrlParamOrderAgainstGolden(t *testing.T) {
 	testParseAgainstGolden(t, "tests/rest_url_params.sysl", "")
 }
 
 func TestRestApi_WrongOrder(t *testing.T) {
-	testParseAgainstPython(t, "tests/bad_order.sysl", "")
+	testParseAgainstGolden(t, "tests/bad_order.sysl", "")
 }
 
 func TestTransform(t *testing.T) {
-	testParseAgainstPython(t, "tests/transform.sysl", "")
+	testParseAgainstGolden(t, "tests/transform.sysl", "")
 }
 
 func TestImpliedDot(t *testing.T) {
-	testParseAgainstPython(t, "tests/implied.sysl", "")
+	testParseAgainstGolden(t, "tests/implied.sysl", "")
 }
 
 func TestStmts(t *testing.T) {
-	testParseAgainstPython(t, "tests/stmts.sysl", "")
+	testParseAgainstGolden(t, "tests/stmts.sysl", "")
 }
 
 func TestMath(t *testing.T) {
-	testParseAgainstPython(t, "tests/math.sysl", "")
+	testParseAgainstGolden(t, "tests/math.sysl", "")
 }
 
 func TestTableof(t *testing.T) {
-	testParseAgainstPython(t, "tests/tableof.sysl", "")
+	testParseAgainstGolden(t, "tests/tableof.sysl", "")
 }
 
 func TestRank(t *testing.T) {
-	testParseAgainstPython(t, "tests/rank.sysl", "")
+	testParseAgainstGolden(t, "tests/rank.sysl", "")
 }
 
 func TestMatching(t *testing.T) {
-	testParseAgainstPython(t, "tests/matching.sysl", "")
+	testParseAgainstGolden(t, "tests/matching.sysl", "")
 }
 
 func TestNavigate(t *testing.T) {
-	testParseAgainstPython(t, "tests/navigate.sysl", "")
+	testParseAgainstGolden(t, "tests/navigate.sysl", "")
 }
 
 func TestFuncs(t *testing.T) {
-	testParseAgainstPython(t, "tests/funcs.sysl", "")
+	testParseAgainstGolden(t, "tests/funcs.sysl", "")
 }
 
 func TestPetshop(t *testing.T) {
 	testParseAgainstPython(t, "petshop.sysl", "../../demo/petshop")
```

Comment:
It's the only file outside the tests/ directory. I could build the .golden. file for it too, but it would belong in the ../../demo/petshop directory, and it's not a good idea for the Makefile to go above its own directory to generate files. I'll fix this up in a subsequent PR. #186